analyses/mutational-signatures/mutational_signatures.Rmd

```diff
@@ -368,6 +369,247 @@ lapply(unique(nature_sigs_primary$short_histology),
 )
 ```
 
+## Determine Signatures de novo  
```

Comment:
My inclination would be to split this out into it's own notebook (+ rename and renumber the notebook you are altering here). My rationale is that this section is thematically different from the original notebook and I'd expect this nice description you have here to be at the top of the notebook. A few other considerations:

- This notebook ends up being over 600 lines long, which can be hard to digest if you're trying to understand what was done (or how this analysis evolved) for the first time
- I believe any _existing_ custom functions that you'd use are in `util/mut_sig_functions.R` and can be sourced into a new notebook
- You have some steps in lines 384-436 that you may want to have some more control over (more thoughts on that below)

Reply:
sounds good